### DISCUSSION TITLE
[Feature Request]: Indicate data point values outside of Y-Bounds

### DISCUSSION THREAD

**Discussion by kylebrandt** [grafana/grafana#36977] [Dashboards, Panels & Visualizations]
Objective: Enable users to be aware of datapoints above (or below, but I mostly would use above) the y limits. This will allow for use of y limits without the concern of a complete lack of awareness of data outside of the plot from the consumer of the visualization.

Suggested Solution: Include two new "Legend Values" (i.e. like min, max, current, etc) which count the number of points that are outside the Y bounds.

Another Idea: Have some sort of visual indicator (i.e. a point with a special shape) that draws between the Title and above the grid. By hovering over it you can get the count of datapoints that are over at that time. If you want to get fancier, you could show a list of the actual values up N values.

Note: I've never seen this sort of visualization before, so don't have any examples to share - hopefully the prose gets the point across.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
